src/app/lib/config/services/thai.js

```diff
@@ -6,6 +6,7 @@ import '@bbc/moment-timezone-include/tz/Asia/Bangkok';
 
 const service = {
   lang: `th`,
```

Comment:
I think we need to add an eslint rule to prevent template literals being used where single quotes should be used.